packages/loading-indicator/src/LoadingIndicator.story.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import { css } from '@leafygreen-ui/emotion';
+import {
+  DarkModeProps,
+  storybookArgTypes,
+  StoryMeta,
+} from '@leafygreen-ui/lib';
+import { spacing } from '@leafygreen-ui/tokens';
+import { Body, InlineCode } from '@leafygreen-ui/typography';
+
+import BlobLoader from './BlobLoader';
+import { Spinner } from '.';
+
+export default StoryMeta({
+  title: 'Components/LoadingIndicator',
+  argTypes: {
+    darkMode: storybookArgTypes.darkMode,
+    description: { control: 'text' },
+  },
+  parameters: {
+    default: 'LiveExample',
+    controls: {
+      exclude: ['className', 'baseFontSize'],
```

Comment:
`className` is included in `storybookExcludedControlParams`

Reply:
why are we excluding baseFontSize?

Reply:
(this should be responsive to baseFontSize)